lib/apis/lists.js

```diff
@@ -0,0 +1,93 @@
+'use strict';
+
+const _ = require('lodash');
+const rp = require('request-promise');
+
+class Lists {
+  constructor(alexaEvent) {
+    this.consentToken = _.get(alexaEvent, 'context.System.user.permissions.consentToken');
+    this.endpoint = 'https://api.amazonalexa.com/v2/householdlists';
+  }
+
+  getOptions(uri = '', method = 'GET', body = {}) {
+    return {
+      uri: `${this.endpoint}/${uri}`,
+      method,
+      body,
+      headers: {
+        Authorization: `Bearer ${this.consentToken}`,
+      },
+      json: true, // Automatically parses the JSON string in the response
+    };
+  }
+
+  getDefaultList(listSuffix) {
+    return rp(this.getOptions())
```

Comment:
async /await ?

Reply:
Travis has a NodeJs 6 test that gets broken when it reads async/await words